lib/appsignal/transaction.ex

```diff
@@ -253,7 +253,19 @@ defmodule Appsignal.Transaction do
   @spec set_sample_data(Transaction.t() | nil, String.t(), any) :: Transaction.t()
   def set_sample_data(nil, _key, _payload), do: nil
 
+  def set_sample_data(%Transaction{} = transaction, "params" = key, payload) do
+    if config()[:send_params] do
+      do_set_sample_data(transaction, key, payload)
```

Comment:
We don't really need a variable for `key` in this clause, as we know its value is always `"params"`. I'd prefer explicitly using the string again in the `do_set_sample_data/3` call for clarity.

Reply:
Will change it then 🙂